pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/admin/UserAdminTestUtil.java

```diff
@@ -42,6 +44,7 @@
  *      Call close()
  */
 @Slf4j
+@Getter
```

Comment:
Why is a getter needed? That changes the contract for this class.

Reply:
I can see why you need the ddpInstanceId and studyGroupId, but I suggest making explicit getters for them, since that limits the contract for this class. IOW, this class is not a bean, so should not have a blanket getters or setters.

Reply:
Understood!